src/Microsoft.IdentityModel.Logging/ISafeLogSecurityArtifact.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.IdentityModel.Logging
+{
+    /// <summary>
+    /// Interface that provides an unsafe method to log a security artifact.
```

Comment:
>Interface [](http://example.com/codeflow?start=8&length=9)

You may want to bring the description you have for the SecurityArtifact class to the interface, this would address the questions that Henrik had here.